src/config/config.js

```diff
@@ -12,3 +12,4 @@ export const DB_NAME = process.env.DB_NAME || ""
 export const DB_PASSWORD = process.env.DB_PASSWORD || ""
 export const DB_HOST = process.env.DB_HOST || "localhost"
 export const DB_PORT = process.env.DB_PORT || 5432
+export const DB_SSL = process.env.DB_SSL || false
```

Comment:
This condition sets DB_SSL = true when .env.DB_SSL exists, regardless of the value of the variable. Env variables are treated as string, so this approach doesn't work

```suggestion
export const DB_SSL = process.env.DB_SSL == "true"
```